config/main.py

```diff
@@ -85,6 +85,7 @@ def set_interface_mode(mode):
     if user != "root":
         bashrc = "/home/{}/.bashrc".format(user)
     else:
+        click.echo("Error: Cannot set interface naming mode for root user!")
```

Comment:
encapsulate 2 statement together into one function, so we can reuse every other place

Reply:
In my latest commit, I have replaced click.echo() + raise click.Abort() calls with one call to click.fail().